flocker/control/test/test_persistence.py

```diff
@@ -38,17 +38,17 @@
     Port, Link, Leases, Lease, BlockDeviceOwnership, PersistentState,
     )
 
-# The UUID values for the Dataset and Node in TEST_DEPLOYMENT match
-# those in the versioned JSON configuration files used by tests in this
-# module. If these values are changed, you will also need to regenerate
-# the test JSON files using the scripts provided in the
-# flocker/control/test/configurations/ directory, using the correct
-# commit checkout to generate JSON appropriate to each config version.
+# The UUID values for the Dataset and Node in the following TEST_DEPLOYMENTs
+# match those in the versioned JSON configuration files used by tests in this
+# module.  If these values are changed, you will also need to regenerate the
+# test JSON files using the scripts provided in the
+# flocker/control/test/configurations/ directory, using the correct commit
+# checkout to generate JSON appropriate to each config version.
 DATASET = Dataset(dataset_id=u'4e7e3241-0ec3-4df6-9e7c-3f7e75e08855',
                   metadata={u"name": u"myapp"})
 NODE_UUID = UUID(u'ab294ce4-a6c3-40cb-a0a2-484a1f09521c')
 MANIFESTATION = Manifestation(dataset=DATASET, primary=True)
-TEST_DEPLOYMENT = Deployment(
```

Comment:
:question: The v1, v2, and v4 scripts won't care about this renaming because they're only ever run from old revisions.